hedera-node/src/main/java/com/hedera/services/ledger/ids/SeqNoEntityIdSource.java

```diff
@@ -50,6 +51,15 @@ public AccountID newAccountId(AccountID sponsor) {
 				.build();
 	}
 
+	@Override
+	public Id newContractId(Id newContractSponsor) {
+		provisionalIds++;
+		return new Id(
+				newContractSponsor.getRealm(),
+				newContractSponsor.getShard(),
+				seqNo.get().getAndIncrement());
```

Comment:
Nit: I'd like to move up line 58-60 by one, so it says "return new Id(new ContractSponsor.getRealm()," with the others on the subsequent lines.